### PR TITLE
UDP-based messages

### PR DESCRIPTION
**Pull request by antoinealb**
This pull request implements UDP messages (one way communication, with no guarantees).

I finished the Python implementation and will start working on the C one. If anyone wants to start reviewing the protocol... :+1:

### CODE REVIEW

**Comment by antoinealb** [comment]
Finished.

Review ? @Stapelzeiger @SyrianSpock ?

File: message.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    size_t name_pos = cmp_mem_access_get_pos(&mem);
+    cmp_mem_access_set_pos(&mem, name_pos + method_name_len);
+    method_name = cmp_mem_access_get_ptr_at_pos(&mem, name_pos);
+
+    /* Do not count message name */
+    argc --;
+
+    for (i = 0; i < callbacks_len; ++i) {
+        if (!strncmp(method_name, callbacks[i].name, method_name_len)) {
+            callbacks[i].cb(argc, &cmp);
```
Comment (Stapelzeiger) [inline message.c]: Break?

Comment (antoinealb) [inline message.c]: not strictly needed, but why not.

**Comment by Stapelzeiger** [comment]
Looks good. I would just add a break when the callback is found.

**Comment by antoinealb** [comment]
Done

**Pull request merged**